DataContext/Context/UAAContext.cs

```diff
@@ -11,8 +11,10 @@ public UAAContext(DbContextOptions<UAAContext> options)
         }
 
         public DbSet<Airline> Airlines { get; set; }
+
         //public DbSet<Airport> Airports { get; set; }
-        //public DbSet<Flight> Flights { get; set; }
+        public DbSet<FlightDc> Flights { get; set; }
```

Comment:
what is Dc?

Reply:
@danielaporcescu Dc stands for _DataContext_. This is because one of the props of this entity has the same name _(Flights)_ and cannot be changed because of the naming in the database

Reply:
NO!

Reply:
Isn't it easier to write DataContext ?